content_scripts/link_hints.coffee

```diff
@@ -167,11 +167,25 @@ class LinkHintsMode
         element.getAttribute("aria-disabled")?.toLowerCase() in ["", "true"])
       return [] # This element should never have a link hint.
 
+    # checks for every valid version of ng-click
+    ngPrefixes = ['', 'data-', 'x-']
+    ngSeparators = ['-', ':', '_']
+    ng = 'ng'
+    click = 'click'
+    hasNgClick = () ->
+      for prefix in ngPrefixes
+        for separator in ngSeparators
+          attr = prefix + ng + separator + click
+          if element.attributes.hasOwnProperty(attr)
+            return true
+      return false
+
     # Check for attributes that make an element clickable regardless of its tagName.
     if (element.hasAttribute("onclick") or
         element.getAttribute("role")?.toLowerCase() in ["button", "link"] or
         element.getAttribute("class")?.toLowerCase().indexOf("button") >= 0 or
-        element.getAttribute("contentEditable")?.toLowerCase() in ["", "contentEditable", "true"])
+        element.getAttribute("contentEditable")?.toLowerCase() in ["", "contentEditable", "true"]) or
+        hasNgClick()
```

Comment:
This function call would be better up by its definition, so it's clear what this check is doing, and that it isn't a criterion for natively clickable elements. Something like `isClickable = true if hasNgClick()` above would do.